python-frontend/typeshed_serializer/resources/custom/aws_cdk/aws_ec2/__init__.pyi

```diff
@@ -25,3 +25,8 @@ class LaunchTemplate(CustomStubBase):
 class SecurityGroup(CustomStubBase):
     connections: Connections
     def add_ingress_rule(peer, connection:Port, description, remote_rule) -> None: ...
+
+class Vpc(CustomStubBase):
+    def select_subnets(self, availability_zones, one_per_az, subnet_filters, subnet_group_name, subnet_name, subnets, subnet_type) -> SelectedSubnets: ...
```

Comment:
Please add `*args` and `*kwargs` arguments. Also please double check return type!